lib/Bio/EnsEMBL/DataCheck/Checks/InterProFeatures.pm

```diff
@@ -0,0 +1,79 @@
+=head1 LICENSE
+
+Copyright [2018] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the 'License');
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an 'AS IS' BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+package Bio::EnsEMBL::DataCheck::Checks::InterProFeatures;
+
+use warnings;
+use strict;
+
+use Moose;
+use Test::More;
+use Bio::EnsEMBL::DataCheck::Test::DataCheck;
+
+extends 'Bio::EnsEMBL::DataCheck::DbCheck';
+
+use constant {
+  NAME        => 'InterProFeatures',
+  DESCRIPTION => 'Check that InterPro data are present and correct',
+  GROUPS      => ['protein_features'],
+  DB_TYPES    => ['core'],
+  TABLES      => ['analysis', 'interpro', 'xref'],
+  PER_DB      => 1
+};
+
+sub tests {
+  my ($self) = @_;
+
+  my $desc_1 = 'interpro table is populated';
```

Comment:
```suggestion
  my $desc_1 = 'Interpro table is populated';
```

Reply:
Well spotted, but I meant to have it lowercase, to match the table name in the db...